backend/sirius-components-emf/src/main/java/org/eclipse/sirius/components/emf/view/diagram/ViewDiagramDescriptionConverterContext.java

```diff
@@ -0,0 +1,54 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.components.emf.view.diagram;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+import org.eclipse.sirius.components.diagrams.description.EdgeDescription;
+import org.eclipse.sirius.components.diagrams.description.NodeDescription;
+import org.eclipse.sirius.components.interpreter.AQLInterpreter;
+
+/**
+ * Represents the context used while converting a View DiagramDescription.
+ *
+ * @author fbarbin
+ */
+public class ViewDiagramDescriptionConverterContext {
+
+    private final Map<org.eclipse.sirius.components.view.NodeDescription, NodeDescription> convertedNodes;
+
+    private final Map<org.eclipse.sirius.components.view.EdgeDescription, EdgeDescription> convertedEdges;
+
+    private final AQLInterpreter interpreter;
+
+    ViewDiagramDescriptionConverterContext(AQLInterpreter interpreter) {
```

Comment:
Why does the constructor has a package visibility? I'll fix it by myself so we can merge this fix this week.

Reply:
my mistake, sorry